ads-txt.php

```diff
@@ -22,6 +22,49 @@
 define( 'ADS_TXT_MANAGER_POST_OPTION', 'adstxt_post' );
 define( 'APP_ADS_TXT_MANAGER_POST_OPTION', 'app_adstxt_post' );
 
+/**
+ * Get the minimum version of PHP required by this plugin.
+ *
+ * @return string Minimum version required.
+ */
+function adstxt_minimum_php_requirement() {
+	return '7.4';
+}
+
+/**
+ * Whether PHP installation meets the minimum requirements
+ *
+ * @return bool True if meets minimum requirements, false otherwise.
+ */
+function adstxt_site_meets_php_requirements() {
+	return version_compare( phpversion(), adstxt_minimum_php_requirement(), '>=' );
+}
+
+// Ensuring our PHP version requirement is met first before loading plugin.
+if ( ! adstxt_site_meets_php_requirements() ) {
+	add_action(
+		'admin_notices',
+		function() {
+			?>
+			<div class="notice notice-error">
+				<p>
+					<?php
+					echo wp_kses_post(
+						sprintf(
+							/* translators: %s: Minimum required PHP version */
+							__( 'Ads.txt requires PHP version %s or later. Please upgrade PHP or disable the plugin.', 'ads-txt' ),
+							esc_html( adstxt_minimum_php_requirement() )
+						)
+					);
```

Comment:
As the translation doesn't include HTML, the kses call can be avoided by escaping the string.

```suggestion
					printf(
						/* translators: %s: Minimum required PHP version */
						esc_html__( 'Ads.txt requires PHP version %s or later. Please upgrade PHP or disable the plugin.', 'ads-txt' ),
						esc_html( adstxt_minimum_php_requirement() )
					);
```

Reply:
Resolved here: 552f5a44499d84f7b312fcf553b7c3b4d696c5b7